modules/cfe_testcase/src/es_misc_test.c

```diff
@@ -28,32 +28,37 @@
  */
 
 #include "cfe_test.h"
+#include "cfe_es_api_typedefs.h"
```

Comment:
Looks like this inclusion is still there.

Reply:
Removed.

Reply:
Added the explicit enum labes.
Cheers Joe.